nixos/modules/programs/x2goserver.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.x2goserver;
+in {
+  options.programs.x2goserver.enable = mkEnableOption "x2goserver";
```

Comment:
No ability to configure at all?

Reply:
I have added some configuration possibilities (debuglevel and nx-agent options), and the config templates should make it easy to add any others if the need arises.
All that stuff goes in `/var/lib/x2go/conf` filled with the default configs, and the files containing the configured settings that are exposed in this module are linked on top.